src/signing/dto/signer-details.dto.ts

```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+
+export class SignerDetailsDto {
+  @ApiProperty({
+    description:
+      'The value to be used in the `signer` field. The format depends on what signer is being used',
```

Comment:
Can we add multiple examples on for each type just for the sake of some clarity?

Reply:
Added on ApiParam